src/SFA.DAS.Funding.SystemAcceptanceTests/SFA.DAS.Funding.SystemAcceptanceTests.csproj

```diff
@@ -16,8 +16,8 @@
     <PackageReference Include="Microsoft.Extensions.Configuration.UserSecrets" Version="6.0.1" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.3.0" />
     <PackageReference Include="RestAssured.Net" Version="1.0.0" />
-    <PackageReference Include="SFA.DAS.Apprenticeships.Types" Version="1.0.3" />
-	  <PackageReference Include="SFA.DAS.CommitmentsV2.Messages" Version="8.29.1" />
+    <PackageReference Include="SFA.DAS.Apprenticeships.Types" Version="1.1.4-prerelease-17" />
```

Comment:
Is the latest version a pre-release? If so we should update that, if not then it can be updated here.

Reply:
packages updated